src/m365/teams/commands/cache/cache-remove.ts

```diff
@@ -71,34 +73,68 @@ class TeamsCacheRemoveCommand extends GraphCommand {
         logger.logToStderr('This command will execute the following steps.');
         logger.logToStderr('- Stop the Microsoft Teams client.');
         logger.logToStderr('- Clear the Microsoft Teams cached files.');
-  
+
         const result = await Cli.prompt<{ continue: boolean }>({
           type: 'confirm',
           name: 'continue',
           default: false,
           message: `Are you sure you want to clear your Microsoft Teams cache?`
         });
-        
+
         if (result.continue) {
           await this.clearTeamsCache(logger);
         }
-      }      
-    } 
+      }
+    }
     catch (err: any) {
       this.handleRejectedODataJsonPromise(err);
     }
   }
 
   private async clearTeamsCache(logger: Logger): Promise<void> {
     try {
-      await this.killRunningProcess(logger);
-      await this.removeCacheFiles(logger);
+      const filePath = this.getFilePath(logger);
+      const folderExists = await this.checkIfCacheFolderExists(filePath, logger);
+      if (folderExists) {
+        await this.killRunningProcess(logger);
+        await this.removeCacheFiles(filePath, logger);
+        logger.logToStderr('Teams cache cleared!');
+      }
+      else {
+        logger.logToStderr('Cache folder does not exist. Nothing to remove.');
+      }
     }
     catch (e: any) {
       throw e.message as string;
     }
+  }
+
+  private getFilePath(logger: Logger): string {
+    const platform = process.platform;
+
+    if (this.verbose) {
+      logger.logToStderr(`Getting filePath of Teams cache folder for platform ${platform}`);
+    }
+
+    let filePath = '';
+
+    switch (platform) {
+      case 'win32':
+        filePath = `${process.env.APPDATA}\\Microsoft\\Teams`;
+        break;
+      case 'darwin':
+        filePath = `${homedir}/Library/Application Support/Microsoft/Teams`;
+        break;
+    }
+    return filePath;
+  }
+
+  private async checkIfCacheFolderExists(filePath: string, logger: Logger): Promise<boolean> {
```

Comment:
This method isn't async because we're using `fs.existsSync` which is sync, so we can simplify it.